docs/reference/migration/apis/deprecation.asciidoc

```diff
@@ -3,6 +3,13 @@
 [[migration-api-deprecation]]
 === Deprecation Info APIs
 
+NOTE: The Deprecation Info API is only guaranteed to be up to date with all
```

Comment:
I'm not 100% sure what the implication is of what you're saying. The deprecation API only returns deperacations up to the version service it. I think out warning should read something like "before upgrading to the next major version please make sure to run this API on the latest minor of the previous version. running it on earlier versions may result in missing crucial information needed for the upgrade"

Reply:
You're right, the wording on this was very confusing.  I've tried to clarify it, but I'm still not completely happy with it.